vm/systemSmartContracts/governance.go

```diff
@@ -210,7 +207,7 @@ func (g *governanceContract) initV2(args *vmcommon.ContractCallInput) vmcommon.R
 //  args.Arguments[2] - minVeto   - 0-10000 - represents percentage
 //  args.Arguments[3] - minPass   - 0-10000 - represents percentage
 func (g *governanceContract) changeConfig(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
-	if !bytes.Equal(g.changeConfigAddress, args.CallerAddr) {
+	if !bytes.Equal(g.ownerAddress, args.CallerAddr) {
```

Comment:
since we can change the proposalFee, I would add also the possibility to change the fee in case the veto did not pass. And add also a sanity check for the not-passed-fee < proposal-fee

Reply:
done